### PR TITLE
CosmosDB dotnet sdk

### PR DESCRIPTION
**Pull request by MehaKaushik**
This PR releases version 1.0.0 for Azure CosmosDB dotnet sdk.

### CODE REVIEW

File: sdk/cosmos-db/Microsoft.Azure.Management.CosmosDB/src/Microsoft.Azure.Management.CosmosDB.csproj [hunk 1]
```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$(RepoEngPath)/mgmt/AzSdk.reference.props" />
+  <!-- Please do not move/edit code below this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.CosmosDB</PackageId>
+    <Description>Provides developers with libraries for the CosmosDB under Azure Resource manager to manage CosmosDB Account, Databases and child resources and their available management capabilities. Create, Delete, Update CosmosDB Account, Databases and child resources. Note: This client library is for CosmosDB under Azure Resource Manager.</Description>
+    <Version>1.0</Version>
```
Comment (isra-fel) [inline sdk/cosmos-db/Microsoft.Azure.Management.CosmosDB/src/Microsoft.Azure.Management.CosmosDB.csproj]:
```suggestion
    <Version>1.0.0</Version>
```

File: sdk/cosmos-db/Microsoft.Azure.Management.CosmosDB/src/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Reflection;
+using System.Resources;
+
+[assembly: AssemblyTitle("Microsoft Azure CosmosDB Management Library")]
+[assembly: AssemblyDescription("Provides management functionality for Microsoft Azure CosmosDB Resources.")]
+
+[assembly: AssemblyVersion("30.0.0.0")]
```
Comment (isra-fel) [inline sdk/cosmos-db/Microsoft.Azure.Management.CosmosDB/src/Properties/AssemblyInfo.cs]:
```suggestion
[assembly: AssemblyVersion("1.0.0.0")]
```

File: sdk/cosmos-db/Microsoft.Azure.Management.CosmosDB/src/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+[assembly: AssemblyFileVersion("30.1.0.0")]
```
Comment (isra-fel) [inline sdk/cosmos-db/Microsoft.Azure.Management.CosmosDB/src/Properties/AssemblyInfo.cs]:
```suggestion
[assembly: AssemblyFileVersion("1.0.0.0")]
```

**Review by isra-fel — CHANGES_REQUESTED** [review]

**Comment by MehaKaushik** [comment]
@isra-fel Made changes as per your suggestions, please have another look.

**Review by isra-fel — APPROVED** [review]

**Pull request merged**